UI testing

Abinash-Anand opened the pull request:
# Re-Design of the website

1. Redesigned the entire website aesthetic.
2. Added a new blog section.
3. Expanded the team section with new additions.
4. Introduced a screening task section.
@Anindyadeep , @Ash-29 
![image](https://github.com/CORD-ai/cord-website/assets/74396889/05cf1455-6354-464b-8938-b87556228b84)

Anindyadeep commented:
@Abinash-Anand looks good, can you remove the merge conflicts, Thanks

Anindyadeep reviewed (changes requested):
Looks good, will review once more once done resolved conflicts

Abinash-Anand commented:
Resolved merge conflicts in index.html, teams.html, projects.html, and stylesheet.

- Kept UI testing changes in index.html
- Incorporated necessary changes in teams.html
- Merged updates in projects.html
- Updated stylesheet with latest design changes

This commit ensures that all conflicts are resolved and the UI testing changes are preserved.
